Pods/CocoaHTTPServer/Core/HTTPConnection.m

```diff
@@ -1036,13 +1036,13 @@ - (HTTPMessage *)newMultiRangeResponse:(UInt64)contentLength
 	// Status Code 206 - Partial Content
 	HTTPMessage *response = [[HTTPMessage alloc] initResponseWithStatusCode:206 description:nil version:HTTPVersion1_1];
 	
-	// We have to send each range using multipart/byteranges
-	// So each byterange has to be prefix'd and suffix'd with the boundry
+	// We have to send each range using multipart/byte ranges
+	// So each byte range has to be prefix'd and suffix'd with the boundary
 	// Example:
 	// 
 	// HTTP/1.1 206 Partial Content
 	// Content-Length: 220
-	// Content-Type: multipart/byteranges; boundary=4554d24e986f76dd6
+	// Content-Type: multipart/byte ranges; boundary=4554d24e986f76dd6
```

Comment:
Oops, I missed that `multipart/byteranges` was a mime type. I'll drop that part throughout this file and wherever this change covers...